packages/react-menu/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@fluentui/react-menu",
+  "version": "0.1.0",
+  "description": "Fluent UI menu component",
```

Comment:
For now can you make it private?

Reply:
It would mean that I have to remove the react-example stub